docs/tutorials/ifci_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,95 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+********************************************
+Exporting IFCI Logs Using QGroundControl
+********************************************
+
+IFCI data can be found in the flight controller logs. These logs can be exported from the flight controller using QGroundControl.
+
+Click the QGroundControl icon on the upper left-hand corner of the application.
+Then click "Analyze Tools" from the pop-up menu.
+
+.. figure:: ../_static/tutorial_images/ifci_log/analyze_tools.png
+    :align: center
+    :alt: Analyze Tools
+
+    QGroundControl Analyze Tools
+
+
+Initially, there may be no logs displayed. If so, click the "Refresh" button to display the logs available on your flight controller.
+
+.. figure:: ../_static/tutorial_images/ifci_log/refresh_button.png
+    :align: center
+    :alt: Refresh Button
+
+    QGroundControl Refresh Logs
+
+
+Then select the log that you want to download, and click the "Download" button. 
+
+.. figure:: ../_static/tutorial_images/ifci_log/download_button.png
+    :align: center
+    :alt: Download Button
+
+    QGroundControl Download Logs
+
+
+You can then choose where to download the selected log file through the file explorer. The file will be downloaded as a .ulg file.
+
+************
+Viewing Logs
+************
+
+There are many ways to view .ulg files. We recommend using the open-source application `PlotJuggler <https://plotjuggler.io/>`_.
+Using your browser, navigate to the `PlotJuggler GitHub page <https://github.com/facontidavide/PlotJuggler>`_ and download the application for your operating system.
+
+Run PlotJuggler once the application is finished installing.
+
+Locate the button to load the flight controller log file. It is located in the upper left-hand corner next to the "Data:" label. When you hover your cursor over the button, the tooltip that appears should display "Load data from file".
+Click the button and select the .ulg flight controller log file.
+
+.. figure:: ../_static/tutorial_images/ifci_log/plotjuggler_load_data_button.png
+    :align: center
+    :alt: Load Data Button
+
+    PlotJuggler Load Data Button
+
+After your log file is loaded, the panel on the left-hand side will be populated with many values.
+
+.. figure:: ../_static/tutorial_images/ifci_log/plotjuggler_loaded_data.png
+    :align: center
+    :alt: Loaded Data 
+
+    PlotJuggler Loaded Data
+
+Locate and select "esc_status". A sub-menu of ESC values will be displayed.
+The values "esc.00", "esc.01", "esc.02", and "esc.03" correspond to the CVI values 0, 1, 2, and 3, respectively.
```

Comment:
This is incorrect. I believe esc.00 corresponds to module ID0, esc.01 to module ID1. CVIs can be whatever as you learned.

@vertiq-jordan If you start your telemetry bitmask at something besides 0, does it still populate based on module ID, or is it offset? For example if you started at module ID 2, would that populate esc.02, or esc.00?